aptos-move/aptos-release-builder/data/release-v1.4.yaml

```diff
@@ -0,0 +1,23 @@
+---
+testnet: true
+remote_endpoint: ~
+is_multi_step: true
+update_sequence:
```

Comment:
Also need the new consensus config for quorum store (if blockchain side is ready to launch) and the feature flags for tx shuffling, and any other missing items on the release sheet

Reply:
Can you help confirm what we need for quorum store? cc @igor-aptos